heft-plugins/heft-typescript-plugin/src/TypeScriptPlugin.ts

```diff
@@ -308,7 +336,7 @@ export default class TypeScriptPlugin implements IHeftTaskPlugin {
 
     if (!partialTsconfigFile) {
```

Comment:
Consider dropping this check.

Reply:
Let's add tsconfig to the "break watch mode" with a comment saying that we should revise that in the future to allow during-runtime modifications to tsconfig.

Reply:
Unfortunately, the name is dynamic, so adding it to the config universally is non-trivial.

Reply:
Leaving a comment, not fixing for now, because the tool doesn't like if you run it without a tsconfig.json. I could detect it being in the list of change files and fail out inside of the TypeScriptPlugin, will consider when fixing solution mode watch mode.